idunn/places/poi.py

```diff
@@ -12,6 +13,19 @@ def __init__(self, d):
             self["properties"] = {p.get("key"): p.get("value") for p in self.get("properties", [])}
         self.properties = self["properties"]
 
+        # If it's a trash can and in Bretagne (in France), we get its volume and the time of the
+        # last update
+        if (self.properties.get("amenity", "") == "recycling"
+            and any(x.get("label", "").lower().endswith("bretagne, france")
+                    for x in d.get("administrative_regions", []))):
+            try:
+                # TODO: use global timeout
+                res = requests.get(f"http://kuzzle:7512/<index>/<collection>/{self.id}", timeout=self.request_timeout)
+               self.properties["trash_volume"] = res.get("result", {}).get("_source", {}).get("volume")
+               self.properties["trash_last_update"] = res.get("result", {}).get("_source", {}).get("hour")
```

Comment:
These new properties are specific to the `recycling` data, so it would probably be cleaner to build them in a new dedicated `RecyclingBlock` that will be necessary at some point to include this data in the Idunn response.